Mage.Sets/src/mage/cards/b/BeneathTheSands.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
Hmm, gen-card.pl doesn't do anything for me, just closes immediately after opening.